packages/mobile/src/exchange/ExchangeTradeScreen.tsx

```diff
@@ -17,10 +17,11 @@ import CeloAnalytics from 'src/analytics/CeloAnalytics'
 import { CustomEventNames, DefaultEventNames } from 'src/analytics/constants'
 import componentWithAnalytics from 'src/analytics/wrapper'
 import { ErrorMessages } from 'src/app/ErrorMessages'
+import { TRANSACTION_MIN_AMOUNT } from 'src/config'
 import { fetchExchangeRate } from 'src/exchange/actions'
 import ExchangeRate from 'src/exchange/ExchangeRate'
 import { ExchangeRatePair } from 'src/exchange/reducer'
-import { CURRENCIES, CURRENCY_ENUM as Token } from 'src/geth/consts'
+import { CURRENCIES, CURRENCY_ENUM as Tokens } from 'src/geth/consts'
```

Comment:
Maybe we should rename CURRENCY_ENUM globally?

Reply:
Yes, considered it but should be a separate PR